public/wp-content/plugins/zzz-imgproxy/imgproxy.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/*
+Plugin Name: zzz-imgproxy
```

Comment:
nejde to nějak upravit aby to nemuselo být zzz a stejně to bylo a po tom s3 pluginu?

Reply:
Imho by bylo lepší tady mít definice funkcí, u kterých je jedno jestli jsou před nebo po pluginu. A ty věci co jsou závislé na tom s3 pluginu tak pustit až v nějakým hooku.